### ISSUE TITLE
Add R 4.0

### ISSUE THREAD

**Issue by kazk** [codewars/runner#5]
Please complete the following information:

- Name: R
- Version: 4.0
- Release Note/Changelog: https://cran.r-project.org/doc/manuals/r-devel/NEWS.html

---

:+1: reaction _might_ help.

**Comment by leechaowen** [comment]
R environment currently has little packages installed. it's better install all packages from CRAN once add R4.0.  It's crazy to request install packages one by one. Do you know what i means?

**Comment by kazk** [comment]
> it's better install all packages from CRAN once add R4.0.

No, we'll only install what makes sense on Codewars. That's why the package request template contains a question why it's useful on Codewars, which you had ignored. Crazy or not, that's how we control our environments.

Remember that Codewars is for learning. When you're building a product, you should use existing packages where it makes sense, but implementing them yourself is a good exercise. Other than that, each package adds possible maintenance cost when we update our environment because packages might have breaking changes that affects our content. A larger container image as a result of more packages also takes longer to build and deploy.
